include/deal.II/meshworker/dof_info.h

```diff
@@ -272,15 +278,25 @@ namespace MeshWorker
 //----------------------------------------------------------------------//
 
   template <int dim, int spacedim, typename number>
-  DoFInfo<dim,spacedim,number>::DoFInfo(const DoFHandler<dim,spacedim> &dof_handler)
+  DoFInfo<dim, spacedim, number>::DoFInfo(const dealii::DoFHandler<dim, spacedim> &dof_handler)
     :
-    level_cell (false)
+    level_cell(false)
```

Comment:
I am confused: shouldn't the number of degrees of freedom per cell change and thus the vector enumerating them be adjusted?

Can you please provide a test that shows that it works the way expected? It can be very simple, say a mesh with 4 cells, 2 different finite elements, say `FE_Q(1)` and `FE_Q(2)` and one vector in the `LocalResults` base class and just report the size of the vector on each cell? I do not see where it is adjusted, but maybe the code is more sophisitcated than I thought.